website/blog/2025-09-15-new-apple-container-extension.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Podman Desktop Apple Container Extension: List Containers & Images on macOS'
+description: Explore the new container extension in Podman Desktop to manage containers and images running with Apple Container on macOS.
+slug: apple-container-extension
+authors: [benoitf]
+tags: [podman-desktop, apple-container, images, containers, macos]
+hide_table_of_contents: false
+image: /img/blog/apple-container-extension/apple-container-extension.png
+---
+
+Apple Silicon macOS users now have a new way to manage containers locally using the [Apple Container](https://github.com/apple/container) project. This tool creates and runs Linux containers inside lightweight virtual machines optimized for Apple silicon, and it's fully written in Swift.
+
+With the new **[Podman Desktop Apple Container extension](https://github.com/podman-desktop/extension-apple-container)** for Podman Desktop, you can now display your containers, images, and logs, all within the Podman Desktop UI 🖥️.
+
+This extension is a technology preview.
+
+![hero](/img/blog/apple-container-extension/apple-container-extension.png)
+
+<!--truncate-->
+
+---
```

Comment:
suggestion: we can probably remove this line separator?

<img width="1009" height="542" alt="Image" src="https://github.com/user-attachments/assets/d3b6f739-e117-415b-94df-702404d130c9" />